.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     name: Run tests
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.6], 3.7]
```

Comment:
Is it means, that tests will be executed on 3.6 only?

Reply:
temp changes, to be reverted